cv32e40s/fv/jaspergold.tcl

```diff
@@ -22,11 +22,25 @@
 proc cvfv_rerun {} {
   clear  -all
 
-  analyze  -sv12  -f  fv.flist
+  # Message Severities
+  ## Error on file not found
+  set_message -error WNL074
+
+  # Analyze & Elaborate
+  analyze  -sv12  -f fv.flist
   elaborate  -top  uvmt_cv32e40s_tb
 
+  # Clock & Reset
   clock  clknrst_if.clk
   reset  ~clknrst_if.reset_n
+
+  # Assumes
+  assume  -from_assert  {*_memory_assert_i.u_assert.a_r_after_a}
+  assume  -from_assert  {*a_xsecure_interface_integrity_obi_*_parity}
+  #TODO:WARNING:silabs-robin parity should be done by obi assertions
+  assume  -from_assert  {*integration_assert_i.a_stable_*}
+  assume  -from_assert  {*integration_assert_i.a_aligned_*}
+  assume  -from_assert  {*integration_assert_i.a_no_scan_cg}
```

Comment:
These assumes are the main part of this PR. They convert an essential few of the asserts into assumes. We need 1) OBI protocol adherence, 2) some stable/aligned address inputs, and 3) no scan testing.